app/views/staff/shared/activities/_tab_nav_item.html.haml

```diff
@@ -0,0 +1,4 @@
+- selected = controller_name.ends_with?(tab)
+%li{class: ["govuk-tabs__list-item", ("govuk-tabs__list-item--selected" if selected)] }
```

Comment:
Tiny comment: would it be neater to add a `.compact` at the end of the array, or does the `nil` class not produce any cruft in the output at all?

Reply:
It actually swallows any `nil`s automatically :)